src/Analysis/GWASV2/Components/Diagrams/PhenotypeHistogram/PhenotypeHistogram.stories.jsx

```diff
@@ -74,15 +74,15 @@ SuccessCase.parameters = {
             ctx.delay(1100),
             ctx.json({
               bins: [
-                { start: 1, end: 10, nr_persons: 100 },
-                { start: 10, end: 20, nr_persons: 200 },
-                { start: 20, end: 30, nr_persons: 300 },
-                { start: 30, end: 40, nr_persons: 400 },
-                { start: 40, end: 50, nr_persons: 500 },
-                { start: 50, end: 60, nr_persons: 400 },
-                { start: 60, end: 70, nr_persons: 350 },
-                { start: 70, end: 80, nr_persons: 100 },
-                { start: 80, end: 90, nr_persons: 50 },
+                {start: 1.4564567,  end: 10.45642, nr_persons: 100},
```

Comment:
I'm seeing NaN in the Storybook xAxis labels but not in the application. I'm not sure if this is a storybook data issue or a potential error. 

<img width="661" alt="image" src="https://user-images.githubusercontent.com/113449836/208949908-daf00277-d8f3-411a-b60e-829b64c682fa.png">

Reply:
I'll leave as this for now. It just means that the current histogram implementation expects numeric labels for the bins on the Xaxis, and this is all we'll have for now. The storybook showing NaN is correct in this case and makes this current limitation more explicit, which is nice IMO.